pulp_rpm/app/advisory.py

```diff
@@ -118,6 +118,8 @@ def resolve_advisories(version, previous_version):
             if pk_to_add:
                 new_advisory = UpdateRecord.objects.filter(pk=pk_to_add[0]).get()
                 tmp_advisories.append(new_advisory)
+            elif new_advisory.pk in pk_to_remove or new_advisory.pk in pk_to_exclude:
+                new_advisory = adv
```

Comment:
So the goal here is, "while comparing dups-with-same-id advisories, if the first-dup  has ended up in to-remove or to-exclude, us the advisory-it-was-compared-against as the one to keep", yeah?   A comment/note at the top of this loop might help future-us remember why we're doing this.

Reply:
done